### PR TITLE
Add depthmapX v.0.8.0

### PR DESCRIPTION
**Pull request by p3palazzo**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew install --cask {{cask_file}}` worked successfully.
- [x] `brew uninstall --cask {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by p3palazzo** [comment]
Ouch. Forgot to leave the PR as a draft until I figured out what test to use. Sorry about that.

**Comment by SMillerDev** [comment]
> GitHub fork (not canonical repository)

Why not use the real upstream?

**Comment by p3palazzo** [comment]
> Why not use the real upstream?

The README of the [upstream repo by Dr Varoudis](https://github.com/varoudis/depthmapX) itself points to the [SpaceGroupUCL fork](https://github.com/SpaceGroupUCL/depthmapX/releases) as the authoritative source for new releases.

File: Casks/depthmapx.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  homepage "https://github.com/SpaceGroupUCL/depthmapX"
+
+  livecheck do
+    url :url
+    strategy :github_latest
+  end
+
+  app "depthmapX.app"
+
+  caveats "depthmapX-cli is not included"
```
Comment (vitorgalvao) [inline Casks/depthmapx.rb:17]: Why not?

Comment (p3palazzo) [reply Casks/depthmapx.rb:17]: @vitorgalvao depthmapX-cli is not required to run depthmapX.app or vice-versa, and is distributed as a separate file in the releases. It should be included separately as a formula, if there is interest in that.

**Review by vitorgalvao — DISMISSED** [review]

**Review by miccal — APPROVED** [review]

**Pull request merged**